db/migrate/20250219085043_remove_currency_from_securities.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveCurrencyFromSecurities < ActiveRecord::Migration[7.2]
+  def change
+    remove_index :securities, :currency
+    remove_column :securities, :currency, :string
+  end
+end
```

Comment:
Would you mind consolidating all of these migrations into one?  Should be able to run `bin/rails db:rollback STEP=8`, delete all but one, consolidate, then run `bin/rails db:migrate` again.

If you want a clean slate, `bin/rails db:migrate:reset` also works.